editoast/src/views/timetable/stdcm.rs

```diff
@@ -245,9 +258,12 @@ async fn stdcm(
 
     // 5. Build STDCM request
     let stdcm_response = STDCMRequest {
+        rolling_stock: PhysicsRollingStock::new(
+            rolling_stock.clone(),
+            stdcm_request.clone().into(),
```

Comment:
Better build the object from the request instead of using `From` to avoid needlessly cloning the whole request.

Moving the `rolling_stock: ...` a few lines below may also help getting rid of the `rolling_stock.clone()`.